aws_lambda_powertools/tracing/base.py

```diff
@@ -5,17 +5,18 @@
 from typing import Any, Generator, List, Optional, Sequence, Union
 
 
+## MAINTENANCE: deprecated, kept for backwards compatibility until v3 is released
 class BaseSegment(abc.ABC):
     """Holds common properties and methods on segment and subsegment."""
 
     @abc.abstractmethod
-    def close(self, end_time: Optional[int] = None):
+    def close(self, end_time: Optional[float] = None):
```

Comment:
Is this a breaking change? Or are we assuming that no one is inheriting from `BaseSegment` today?

Reply:
We will deprecate this in Powertools v3. We're not using this class today, but we never know if customers are, so it's best to revert this change.

Reverted.